src/web/package.json

```diff
@@ -7,24 +7,40 @@
     "dev": "vite",
     "build": "tsc -b && vite build",
     "lint": "eslint .",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "test": "jest"
   },
   "dependencies": {
+    "@fluentui/react": "^8.121.6",
+    "@fluentui/react-components": "^9.55.1",
+    "autoprefixer": "^10.4.20",
     "eslint-plugin-react": "^7.37.1",
     "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "react-icons": "^5.3.0",
```

Comment:
What icons are we using here? Can we get what we need from Fluent UI?

Reply:
I will look for equivalent in fluent ui. I started converting these components first and hence didn't check yet. But I will definitely work on this and find equivalent icons.

Reply:
removed react icons package, using only fluent v2 icons